config/default/manager_auth_proxy_patch.yaml

```diff
@@ -20,6 +26,12 @@ spec:
         - containerPort: 8443
           name: https
       - name: manager
+        securityContext :
+          allowPrivilegeEscalation: false
+          runAsNonRoot: true
+          capabilities:
+            drop:
+              - all
```

Comment:
the manager has already securityContext defined here:
https://github.com/codeready-toolchain/member-operator/blob/ec800b33605d265754c7791b6345cc229f62a27e/config/manager/manager.yaml#L36-L37
could we move the config there?

Reply:
same as the other comment, the security context of container overrides the pod level security context so i think we should keep it at container level to avoid vulnerability